Doc/whatsnew/3.14.rst

```diff
@@ -708,6 +708,17 @@ asyncio
   reduces memory usage.
   (Contributed by Kumar Aditya in :gh:`107803`.)
 
+
+base64
+------
+
+* Improve performance of :func:`base64.b16decode` by a factor ten by
+  removing an unnecessary regular expression. Consequently, :mod:`re`
+  is no more implicitly available as ``base64.re`` and import time of
+  :mod:`base64` is improved by a factor six.
+  (Contributed by Bénédikt Tran in :gh:`118761`.)
```

Comment:
'factor ten' doesn't make sense (it should be 'factor of ten'), but here's a suggestion:

```suggestion
* Improve the performance of :func:`base64.b16decode` by up to ten times,
  and reduce the import time of :mod:`base64` by up to six times.
  (Contributed by Bénédikt Tran, Chris Markiewicz, and Adam Turner in :gh:`118761`.)
```

I've also suggested crediting me and Chris, though feel free to remove - it wasn't in any of your original drafts.

Reply:
I'm perfectly fine crediting everyone!

Reply:
> factor ten' doesn't make sense (it should be 'factor of ten'), 

Sorry, this is me translating from French to English!